src/wp-includes/class-wp-theme-json.php

```diff
@@ -2886,4 +2907,122 @@ public function get_data() {
 		return $output;
 	}
 
+	/**
+	 * Sets the spacingSizes array based on the spacingScale values from theme.json.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @return void
```

Comment:
Return should be `null` also.
```suggestion
	 * @return null|void
```